libnavigation-core/src/test/java/com/mapbox/navigation/core/navigator/LocationExTest.kt

```diff
@@ -103,7 +120,7 @@ class LocationExTest {
             NULL_VALUE,
             NULL_VALUE,
             NULL_VALUE,
-            EMPTY_EXTRAS,
+            EXTRAS,
```

Comment:
This test has a name `checkLocationWithZeroParams`. Not empty `EXTRAS` doesn't match the tests's name.